DasharoPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.c

```diff
@@ -1067,6 +1070,168 @@ WarnIfRecoveryBoot (
   BootLogoEnableLogo ();
 }
 
+
+
+typedef struct {
+  TPM_ALG_ID AlgId;
+  CHAR16    *Name;
+} HASH_ALG_ENTRY;
+
+STATIC CONST HASH_ALG_ENTRY mHashAlgs[] = {
+  { TPM_ALG_SHA1,    L"SHA1"    },
+  { TPM_ALG_SHA256,  L"SHA256"  },
+  { TPM_ALG_SHA384,  L"SHA384"  },
+  { TPM_ALG_SHA512,  L"SHA512"  },
+  { TPM_ALG_SM3_256, L"SM3_256" },
+};
+
+STATIC
+VOID
+WarnIfSinglePCRBank (
+  VOID
+)
+{
+  EFI_STATUS     Status;
+  EFI_EVENT      Events[1];
+  BOOLEAN        CursorVisible;
+  UINTN          CurrentAttribute;
+  UINTN          Index;
+  EFI_INPUT_KEY  Key;
+  UINT32         TpmHashAlgorithmBitmap;
+  UINT32         ActivePcrBanks;
+  UINT32         RequestedActivePcrBanks;
+  UINTN          Size;
+  UINTN          i;
+  UINTN          OptionCount = 0;
+  UINTN          AvlIdx[ARRAY_SIZE(mHashAlgs)] = {0};
+  CHAR16         OptLine[81];
+  UINT32         SelectedMask;
+
+  Status = Tpm2GetCapabilitySupportedAndActivePcrs(&TpmHashAlgorithmBitmap, &ActivePcrBanks);
+  if (EFI_ERROR(Status)) {
+      return;
+  }
+  Size = sizeof(RequestedActivePcrBanks);
+  Status = gRT->GetVariable(
+            REQUESTED_ACTIVE_PCR_BANKS_VARIABLE_NAME,
+            &gEfiTcg2PhysicalPresenceGuid,
+            NULL,
+            &Size,
+            &RequestedActivePcrBanks
+            );
+  // 
+  // If the variable doesn't exist, there's been no request to change the
+  // PCR banks.
+  // 
+  if (Status == EFI_NOT_FOUND) {

```

Comment:
```suggestion
  if (EFI_ERROR(Status)) {
```
since we don't have any data to use in case of other errors as well.

Reply:
[Changed](https://github.com/Dasharo/edk2/pull/282/files#diff-306afacd6b8e1d90f20c52fe64cf2eab3b6d949cdf5225d2b310a06c84a950bcR1122)